### PR TITLE
API-5282 smart-cards fhir/v0/r4 rule

### PR DESCRIPTION
**Pull request by jblefkove-va**
Is this a new product? no

Has this been deployed successfully from a branch? yes

Are load balancer rules being removed or renamed? no

Are ingress definitions being removed or renamed? yes

Are Kubernetes namespaces being removed or renamed? no

Has DevOps been consulted on resource limits? n/a

### CODE REVIEW

File: products/smart-cards.yaml [hunk 1]
```diff
@@ -16,6 +16,23 @@ metadata:
...
+  name: smart-cards-fhir-ingress
+  namespace: $DU_NAMESPACE
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+spec:
+  rules:
+    - http:
+        paths:
+          - path: /fhir/v0/(.*)
```
Comment (jhulbert-va) [inline products/smart-cards.yaml:29]: Not sure I'm a fan. This is going to catch _everything_ on the `/fhir/v0` path if it wasn't matched to anything previously (I think that's only DQ).

Comment (jblefkove-va) [reply products/smart-cards.yaml:29]: how about `/fhir/v0/r4/Patient/(.*)`

Comment (jhulbert-va) [reply products/smart-cards.yaml:29]: Will the path always end in `$HealthWallet.issueVc` for this path?

Comment (jblefkove-va) [reply products/smart-cards.yaml]: Path is fully qualified now

File: products/smart-cards.yaml [hunk 2]
```diff
@@ -16,6 +16,23 @@ metadata:
...
+  name: smart-cards-fhir-ingress
+  namespace: $DU_NAMESPACE
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+spec:
+  rules:
+    - http:
+        paths:
+          - path: /fhir/v0/(r4/Patient/[A-Z0-9]+/[$]HealthWallet.issueVc)
```
Comment (jhulbert-va) [inline products/smart-cards.yaml:29]: You'll probably want `[.]` as well. Right now, the `.` is going to match any character.

**Review by jhulbert-va — APPROVED** [review]
LGTM

**Comment by jblefkove-va** [comment]
This repo does not have squash and merge, so I did a force push to clean up the history.

**Review by jhulbert-va — APPROVED** [review]
:shipit:

**Pull request merged**